resources/sass/_multiselect.scss

```diff
@@ -21,4 +21,9 @@
 // User list tag item
 .multiselect-tag-user {
   background-color: rgba(16, 185, 129, var(--bs-bg-opacity, 1));
+}
+
+.input-group .multiselect {
+  width: auto;
+  flex: 1;
 }
```

Comment:
what is this for?

Reply:
probably because of the delimiter field. But had no effect on it. Removed it in eb84b1f1998e5206e0b02daba3206357f3a862ff